### ISSUE TITLE
Node's coordinates in the SOM

### ISSUE THREAD

**Issue by JessitaMS** [fcomitani/simpsom#20]
Hi @fcomitani:

I have some problems to determinate the node's coordinate in the bidirectional map. If I took, the trainned net and I want to see the centroids of the node, only I need to do is:

#SOM 4X5
net45 = sps.somNet(4, 5, data, PBC=True)
#Train the network for 10000 epochs and with initial learning rate of 0.01. 
s45 = net45.train(0.01, 10000)

#Visualize the centroids of each features into the node 0
s45[0]

![s45](https://user-images.githubusercontent.com/26332548/121863054-baeec580-ccfb-11eb-8515-299908332d98.JPG)

But if I want only two coordinates from where is located in the MAP this node, I can´t do this. So, can you help me to find where I get this two values that I need, 'cause I review your code and I don't find it.

Thanks so much,

**Comment by fcomitani** [comment]
Hi @JessitaMS,

The nodes are stored in the `nodeList` attribute of your net object, to see where each node is located (in the 2d hexagonal map) you'll need the `.pos` attribute of each node.

So for example in your case, to find the coordinates of the first node you'll need 

`net45.nodeList[0].pos`

I'm going to add a note about this in the README, since it's basic information and it should be easy to find.
Thanks for bringing this up.

Hope this helps!

**Comment by fcomitani** [comment]
I'm closing this issue. Please feel free to reopen it if anything else comes up.

**Comment by JessitaMS** [comment]
> Hi @JessitaMS,
> 
> The nodes are stored in the `nodeList` attribute of your net object, to see where each node is located (in the 2d hexagonal map) you'll need the `.pos` attribute of each node.
> 
> So for example in your case, to find the coordinates of the first node you'll need
> 
> `net45.nodeList[0].pos`
> 
> I'm going to add a note about this in the README, since it's basic information and it should be easy to find.
> Thanks for bringing this up.
> 
> Hope this helps!


I'm sorry for my reply delayed. Thanks so much for you answer. I'll try to used it! :D
